src/UI/templates/default/Button/button.less

```diff
@@ -2,4 +2,41 @@ a.btn-link.ilSubmitInactive, a.btn-link.ilSubmitInactive:hover {
 	background-color: inherit;
 	color: @text-color;
 	padding: 0;
-}
\ No newline at end of file
+}
+
+.btn-tag {
+	&.btn-tag-inactive {
+		cursor: default !important;
+	}
+	&.btn-tag-relevance-verylow {
+		background-color: desaturate(@btn-tag-bg, 80%);
+		color: contrast(desaturate(@btn-tag-bg, 80%));
```

Comment:
@Amstutz: Is this `desaturate` something we want to have? I vaguely remember some discussion with @catkrahl about problems that arise due to this.